apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/NormalizedCacheStore.java

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+
+public abstract class NormalizedCacheStore {
```

Comment:
It almost feels like we want to return an absent optional here. Or something similar. So when reading it can return an empty container or something.